Remove meta data from add event form.

ObothA opened the pull request:
# Summary of changes
- Create a separate form to add an event without meta data.

# How to test
- Clone the repository and checkout to bug/add-event branch.
- Navigate to the calendar.
- Click Create Event and fill out the form.

![image of form](https://user-images.githubusercontent.com/38153543/167431992-7a7882a0-24a9-4569-acc4-95419b869b4f.png)

androidcode33 commented on package.json:
```diff
@@ -71,7 +71,7 @@
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
     "react-router-redux": "^4.0.8",
-    "react-scripts": "^4.0.3",
```
@ObothA lets retainer the old version "react-scripts": "^4.0.3". Remove the package.json in your comment. The packages should not be updated.

ObothA replied:
@androidcode33 from a fresh install, react scripts had other packages with which dependencies were clashing. With npm it couldn't run at all, yarn tried to fix the issues but mouse events were disabled making the project unusable. So upgrading react scripts fixed that issue. You can try running a fresh setup of the worship harvest branch to confirm. In light of that information, should we maintain the upgrade or we revert?

androidcode33 replied:
@ObothA that was on your machine and someone else won't get the same issues. Just ignore the package.json file

ObothA replied:
Alright.

androidcode33 reviewed (commented):
@ObothA fix this item package.json

androidcode33 commented:
@ObothA You left package-lock.json. Remove it as well.

ObothA commented:
> @ObothA You left package-lock.json. Remove it as well.

Done.

ObothA commented:
> @ObothA You left package-lock.json. Remove it as well.

Now that I think about it, won't removing it altogether delete the one in the source repo! How about generating a new
 package-lock.json with the correct versions of react-scripts?

androidcode33 commented:
You are supposed to just remove it from what is committed, not deleting the files themselves, what did you do?

ObothA commented:
> You are supposed to just remove it from what is committed, not deleting the files themselves, what did you do?

@androidcode33 Initially I had deleted from my machine. I have now just removed it. It's no longer available in the files changed.
